evm/src/cpu/kernel/asm/sha2/temp_words.asm

```diff
@@ -0,0 +1,30 @@
+%macro sha2_temp_word1
+    // stack: e, f, g, h, K[i], W[i]
+    DUP1
+    // stack: e, e, f, g, h, K[i], W[i]
+    %sha2_bigsigma_1
+    // stack: Sigma_1(e), e, f, g, h, K[i], W[i]
+    %stack (sig, (efg: 3)) -> (efg, sig)
```

Comment:
nit: maybe best to just write out `e, f, g` to avoid any possible confusion. I.e. readers might think that `efg` is some cohesive structure, or might be thrown off by the similarity of the `efg` and `sig` names.